### PR TITLE
Update parts of viz-stack to latest versions

### PR DESCRIPTION
**Pull request by tribal-tec**
…ions

### CODE REVIEW

File: bbp/viz/rtneuron/default.nix [hunk 1]
```diff
@@ -40,7 +40,7 @@ stdenv.mkDerivation rec {
   src = fetchgitPrivate {
     url = config.bbp_git_ssh + "/viz/RTNeuron";
     rev = "47dce1e11a6534dc2eb4971a589cec459c9cc274";
-    sha256 = "04215jjjl1sqc6lna19bd8zbyl9r89snjgysp7iwrmd9mj5x53wl";
+    sha256 = "14215jjjl1sqc6lna19bd8zbyl9r89snjgysp7iwrmd9mj5x53wl";
```
Comment (tristan0x) [inline bbp/viz/rtneuron/default.nix]: Hey @tribal-tec 
This is this change that breaks the build. Can you revert it please?

Comment (tribal-tec) [reply bbp/viz/rtneuron/default.nix]: ah sure, this shouldn't be here

File: bbp/viz/brayns/default.nix [hunk 2]
```diff
@@ -5,7 +5,8 @@
 , boost
 , assimp ? null
 , ospray
-, embree 
+, embree
```
Comment (tristan0x) [inline bbp/viz/brayns/default.nix]: I think `embree` is not used anymore. Can you get rid of it?

**Comment by tristan0x** [comment]
This repository's history thanks you @tribal-tec for rebasing your changes :smile_cat:

**Comment by tristan0x** [comment]
```
[ 91%] Building CXX object brain/python/CMakeFiles/brain_python.dir/spikeReportReader.cpp.o
g++: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <http://gcc.gnu.org/bugs.html> for instructions.
```

wow!

**Comment by tristan0x** [comment]
retest this please

**Pull request merged**